cli/src/subcommand.rs

```diff
@@ -59,6 +61,9 @@ pub enum Command {
         /// --exclude=require-concurrent-index-creation,ban-drop-database
         #[structopt(short, long, use_delimiter = true)]
         exclude: Option<Vec<RuleViolationKind>>,
+        /// Exits with an error code when specified
+        #[structopt(long)]
+        exit_on_error: bool,
```

Comment:
thoughts on calling it something like:

```suggestion
        exit_on_violations: bool,
```

or maybe

```suggestion
        exit_with_code_on_violations: bool,
```

since we do exit with an error already, just not when there's violations?

Reply:
Which one do you prefer? Another option on the table would be instead of using `exit` using something like `fail`: `fail_on_violations`

Reply:
Oh I like `fail_on_violations`!!